### DISCUSSION TITLE
Option to display related library items within Document Sets

### DISCUSSION THREAD

**Discussion by PowerProvoker** [microsoft-search/pnp-modern-search#2763] [Ideas]
Hello, folks

I have a client request where we are displaying items in a Results part from Document Sets. Currently, if you display document sets from a SharePoint library - you cannot determine what is in those document sets. If you click on returned document set, you are taken directly to that document set and away from the search results.

Is there already a way in which we can display documents contained within a document set underneath a Document Set result using a handlebar expression perhaps? Would be a great feature for those looking to return Folders and then documents held within those folders for context.

**Comment by kasperbolarsen** [comment]
Perhaps  https://github.com/microsoft-search/pnp-modern-search/discussions/2754 can point you in the right direction? 🙂

**Comment by PowerProvoker** [comment]
> Perhaps #2754 can point you in the right direction? 🙂

This looks what I'm after! The <pnp-collapsible> component definitely fits the style of what I'm after - now I'm just not aware of how to capture the content from within a Document Set. Document Set would be the 'Data-Group-Name' element in my case, and then the collapsible content within would be the documents... I'm just not sure on how to capture that Parent-Child relationship in Handlebar expressions.
